### PR TITLE
Enhanced documentation with Simulate Annealing Matrix Benchmarking.

### PR DESCRIPTION
**Pull request by Blackjack92**

### CODE REVIEW

**Comment by Blackjack92** [comment]
Sry the formatting is not the expected style ...

**Comment by ge0ffrey** [comment]
@Blackjack92 Due to the formatting change (from 120 chars per line to 80 chars per line apparently), it's impossible for me to see the changes and review/merge this PR. Sorry :/

**Comment by Blackjack92** [comment]
Yes I will change it. Yesterday I had no time left :)

**Comment by Blackjack92** [comment]
@ge0ffrey Can you check the new commit?

File: optaplanner-docs/src/main/docbook/en-US/Chapter-Benchmarking_and_tweaking/Chapter-Benchmarking_and_tweaking.xml [hunk 1]
```diff
@@ -1078,8 +1082,32 @@
...
+    &lt;/solver&gt;
+  &lt;/solverBenchmark&gt;
+&lt;/#list&gt;
+&lt;/#list&gt;
 &lt;/plannerBenchmark&gt;</programlisting>
 
+      <note>
+        <para>When using Simulated Annealing Matrix Benchmarking on Windows it is necessary to replace the "/" from the
+        <literal>startingTemperature</literal> (<literal>?replace("/", "_")</literal>) so that it is not part of the
+        file name. Otherwise there is a problem with the naming conventions of files on Windows.</para>
```
Comment (ge0ffrey) [inline optaplanner-docs/src/main/docbook/en-US/Chapter-Benchmarking_and_tweaking/Chapter-Benchmarking_and_tweaking.xml]: It's not just on windows: SolverBenchmark's name is validated against an regex that doesn't allow /, so it fails the same on all platforms.

I 'll merge and adjust that :)

File: optaplanner-docs/src/main/docbook/en-US/Chapter-Benchmarking_and_tweaking/Chapter-Benchmarking_and_tweaking.xml [hunk 2]
```diff
@@ -1078,8 +1082,32 @@
...
 &lt;/#list&gt;
+
+&lt;#list [1, 100, 1000] as acceptedCountLimit&gt;
+&lt;#list [1hard/5medium/10soft, 1hard/10medium/20soft, 1hard/20medium/50soft, 1hard/50medium/70soft] as startingTemperature&gt;
+  &lt;solverBenchmark&gt;
+    &lt;name&gt;SimulatedAnnealing acceptedCountLimit ${acceptedCountLimit} temperature ${startingTemperature?replace("/", "_")}&lt;/name&gt;
+    &lt;solver&gt;
+      &lt;localSearch&gt;
+        &lt;acceptor&gt;
+          &lt;simulatedAnnealingStartingTemperature&gt;${startingTemperature?replace("/", "_")}&lt;/simulatedAnnealingStartingTemperature&gt;
```
Comment (ge0ffrey) [inline optaplanner-docs/src/main/docbook/en-US/Chapter-Benchmarking_and_tweaking/Chapter-Benchmarking_and_tweaking.xml]: in this element there shouldn't be a replace.

Did you run this code? I 'll try to run it and fix it.

**Comment by ge0ffrey** [comment]
Thanks :) Merged and I added a commit on top of it (see comments for detail)

**Comment by Blackjack92** [comment]
Thanks for the comments. (I run the code above)

**Pull request merged**